playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx

```diff
@@ -85,10 +85,11 @@ const TextInput = (props: TextInputProps, ref: React.ElementRef<"input">) => {
   )
   const textInput = (
     <input
-        {...props}
+        {...domSafeProps(props)}
         className="text_input"
         disabled={disabled}
         id={id}
+        key={id}
```

Comment:
I saw the warning `Warning: Each child in a list should have a unique "key" prop.` still in the jest ci tests. @codemonium figured out a way to at least suppress that warning!

```jsx
<If condition={addOnAlignment == 'left'}>
  <>
    <Card
        className={`${addOnDarkModeCardCss} add-on-card card-left-aligned`}
        dark={dark}
    >
      {addOnIcon}
    </Card>
    {textInput}
  </>
  <Else />
  <>
    {textInput}
    <Card
        className={`${addOnDarkModeCardCss} add-on-card card-right-aligned`}
        dark={dark}
    >
      {addOnIcon}
    </Card>
  </>
</If>
```

Reply:
Maybe that's better/easier than setting a default/uniq id for each TextInput as I mentioned in a previous comment.